database/insertData/01_permission_policies.js

```diff
@@ -95,16 +138,26 @@ exports.queries = [
       }
     }
   }`,
-  // basicAssign
+  // assignBasic
   `mutation createPolicy {
     createPermissionPolicy(
       input: {
         permissionPolicy: {
-          name: "basicAssign"
+          name: "assignBasic"
           rules: {
             application: {
               view: { template_id: "jwtPermission_bigint_templateId" }
             }
+            review: {
+              view: {
+                reviewer_id: "jwtUserDetails_bigint_userId"
```

Comment:
Shouldn't be `assigner_id` instead? Also wouldn't all reviews be visible to Assigner that have permission to assign? So not sure how to restrict visible review/review_assignment to them. Because it would be either if: 
- assigned by this user
- Not fully assigned and user has permission to assign template.

Reply:
Yeah, I'm not sure either. I think for assigner we just want to restrict access to review_assignment by template only (so they can see all the assignment possibilities). But because `template_id` is not a field on review_assignment, it'll probably have to be one of those nested queries Andrei was talking about. Will figure it out. Thanks.

Reply:
create check file, and run it with ts-node
```
import { getSqlConditionFromJSON } from './helpersUtilities'
console.log(
  getSqlConditionFromJSON({
    reviewer_id: {
      $in: {
        $select: {
          id: true,
          $from: 'application',
          $where: { template_id: 'jwtPermission_bigint_templateId' },
        },
      },
    },
  })
)
```

Reply:
Okay, _finally_ managed to get this insert to work. Put the rules into a variable as per @andreievg 's suggestion, and updated insertData.js to look for queries containing variables (by checking for Object or string)

ee29828a

Reply:
🎉 🎉 🎉 🎉 🎉